code/game/objects/items/RCD.dm

```diff
@@ -550,9 +558,6 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 	if(!do_after(user, delay, target = A))
 		qdel(rcd_effect)
 		return FALSE
-	if(!checkResource(rcd_results["cost"], user))
-		qdel(rcd_effect)
-		return FALSE
```

Comment:
This will allow people to start several constructions simultaneously and have them succeed while only having the resources for one.
Don't remove this check